city_scrapers/spiders/ind_school_board.py

```diff
@@ -87,12 +91,17 @@ def _parse_start(self, item):
         """Parse start datetime as a naive datetime object."""
 
         title_str = item.xpath("./name/text()").extract_first()
-        time_str = "12:00 AM"
+        print(title_str)
```

Comment:
Watch out for stray `print` statements. I know it's easy to forget about these. Would recommend searching the repo and cleaning up any stragglers 🧹